modules/monitoring-creating-alert-routing-for-user-defined-projects.adoc

```diff
@@ -21,6 +13,7 @@ If you are a non-administrator user who has been given the `alert-routing-edit`
 
 * A cluster administrator has enabled monitoring for user-defined projects.
 * A cluster administrator has enabled alert routing for user-defined projects.
+* A cluster administrator has enabled Alertmanager for user-defined projects.
```

Comment:
As written elsewhere, there could be a dedicated Alertmanager for user-defined alerts in the `openshift-user-workload-monitoring` namespace or we reuse the instance existing in the `openshift-monitoring` namespace.

Reply:
I've deleted this as a prerequisite.